package.json

```
@@ -8,8 +8,8 @@
   "scripts": {
     "lint": "standard | snazzy",
     "lint-ci": "standard",
-    "test": "tap --no-cov",
-    "test-ci": "tap --cov --no-check-coverage --coverage-report=text",
+    "test": "borp -p \"test/**/*.js\"",
+    "test-ci": "borp --coverage -p \"test/**/*.js\"",
```

Utilizing single quotes will prevent shell expansion and provide the glob directly to the runner tool:

```suggestion
    "test": "borp -p 'test/**/*.js'",
    "test-ci": "borp --coverage -p 'test/**/*.js'",
```

@mcollina does your tool support platform agnostic glob expansion?